crates/wdk-build/src/cargo_make.rs

```diff
@@ -20,6 +20,13 @@ use crate::{
 };
 
 const PATH_ENV_VAR: &str = "Path";
+const WDK_VERSION_ENV_VAR: &str = "WDK_VER";
```

Comment:
Is `WDK_VER` an environment variable thats typically already exposed in the eWDK prompt, but we don't expose in WDR cargo make? If its not a "standard" WDK env var name, I think we should name this as `WDK_BUILD_DETECTED_WDK_VERISON` or similar.